### PR TITLE
MklEltwiseFwdPrimitive implemented twice (same class name used)

### PR DESCRIPTION
**Pull request by milpuz01**
In files tensorflow/core/kernels/mkl/mkl_eltwise_activation_base_op.h and tensorflow/core/kernels/mkl/mkl_relu_op.cc there are two classes with the same name: MklEltwiseFwdPrimitive. We have noticed that this can lead to segmentation fault at runtime when method is called on different instantiation of the class then what is expected (i.e calling Execute() on implementation in mkl_eltwise_activation_base_op.h instead of one in mkl_relu_op.cc). Since implementation are slightly different this patch renames MklEltwiseFwdPrimitive in mkl_eltwise_activation_base_op.h to MklEltwiseFwdActivationPrimitive to avoid the issue.

### CODE REVIEW

**Comment by gbaned** [comment]
Hi @penpornk Can you please review this PR ? Thank you!

**Comment by sachinmuradi** [comment]
@milpuz01 Thank you for the changes, looks good to me.
- Sachin from Intel-Tensorflow team

File: tensorflow/core/kernels/mkl/mkl_eltwise_activation_base_op.h [hunk 1]
```diff
@@ -40,21 +40,21 @@ using dnnl::memory;
...
 using dnnl::stream;
 
-using EltwiseFwdPd = dnnl::eltwise_forward::primitive_desc;
+using EltwiseFwdActivationPd = dnnl::eltwise_forward::primitive_desc;
 
 namespace tensorflow {
 
 template <typename T>
-class MklEltwiseFwdParams {
+class MklEltwiseFwdActivationParams {
```
Comment (penpornk) [inline tensorflow/core/kernels/mkl/mkl_eltwise_activation_base_op.h:48]: It's not just the class name. The implementation of this class is very similar to `MklEltWiseFwdParams` in `mkl_relu_op.cc`. I'm okay with renaming as a quick fix for TF 2.10 cherry-pick. But long-term I think we should just consolidate them to one class.

Please add this comment in the code (please wrap the line if it's longer than 80 characters):
```
// TODO(tf-onednn): Consolidate this class with `MklEltWiseFwdParams` in `mkl_relu_op.cc`.
```

Thank you very much!

**Review by penpornk — CHANGES_REQUESTED** [review]
@milpuz01 Thank you for the fix!
@sachinmuradi Thank you for the quick review!

File: tensorflow/core/kernels/mkl/mkl_eltwise_activation_base_op.h (hunk 1 above)
Comment (milpuz01) [reply tensorflow/core/kernels/mkl/mkl_eltwise_activation_base_op.h:53]: Thanks @penpornk and @sachinmuradi. Added comment as requested.

**Review by penpornk — APPROVED** [review]
Thank you for the changes!

**Pull request merged**